types/mocha/index.d.ts

```diff
@@ -391,6 +392,17 @@ declare namespace Mocha {
         function exports(suite: Suite): void;
     }
 
+    namespace nodejs {
+        /**
+         * This Runner delegates tests runs to worker threads. Does not execute an Runnables by itself!
+         *
+         * @see https://mochajs.org/api/module-parallel-buffered-runner-parallelbufferedrunner
+         */
+        class ParallelBufferedRunner extends Runner {
+            workerReporter(path: string): this;
+        }
+    }
+
```

Comment:
```suggestion
```

There isnt class `nodejs.ParallelBufferedRunner `

Reply:
But we have exported [`lib/nodejs/ParallelBufferedRunner` class](https://github.com/mochajs/mocha/blob/master/lib/nodejs/parallel-buffered-runner.js)